pkg/buildkite/pipelines.go

```diff
@@ -383,6 +389,36 @@ func CreatePipeline(client PipelinesClient) (tool mcp.Tool, handler mcp.TypedToo
 				return mcp.NewToolResultError(err.Error()), nil
 			}
 
+			// create webhooks by default to align with the behavior in the dashboard
+			createWebhook := true
+			if args.CreateWebhook != nil {
+				createWebhook = *args.CreateWebhook
+			}
+
+			if createWebhook {
+				_, err := client.AddWebhook(ctx, args.OrgSlug, pipeline.Slug)
+				if err != nil {
+					result := map[string]any{
+						"pipeline": pipeline,
+						"webhook": map[string]any{
+							"created": false,
+							"error":   err.Error(),
+							"note":    "Pipeline created successfully, but webhook creation failed.",
+						},
+					}
+					return mcpTextResult(span, &result)
+				}
+
+				result := map[string]any{
+					"pipeline": pipeline,
+					"webhook": map[string]any{
+						"created": true,
+						"note":    "Pipeline and webhook created successfully.",
+					},
+				}
+				return mcpTextResult(span, &result)
+			}
+
 			return mcpTextResult(span, &pipeline)
```

Comment:
I would just have one result for the pipeline create, with webhook not set when it is set to false. 

Would be easier to declare a struct for this result as it is then extensible later.

Reply:
Should be doable! I'll give that a try.

Reply:
@wolfeidau This look a bit better? https://github.com/buildkite/buildkite-mcp-server/pull/160/commits/2b3a7c5fbfd7712a301a676008140a40709e2cc0